hosts/juno/junocam.py

```diff
@@ -61,23 +60,25 @@ def from_file(filespec, fast_distortion=True,
         fmeta = Metadata(flabels[i])
 
         if snap:
-            item = (oops.obs.Snapshot(("v","u"),
-                                 (fmeta.tstart, fmeta.tdi_texp), fmeta.fov,
-                                 "JUNO", "JUNO_JUNOCAM",
-                                 instrument = "JUNOCAM",
-                                 filter = fmeta.filter,
-                                 data = framelets[:,:,i]))
+            item = (oops.obs.Snapshot(
+                ("v","u"),
+                (fmeta.tstart, fmeta.tdi_texp), fmeta.fov,
+                "JUNO", "JUNO_JUNOCAM",
+                instrument = "JUNOCAM",
+                filter = fmeta.filter,
+                data = framelets[:,:,i]))
```

Comment:
This indentation is very hard to read. It wasn't obvious that this was a Snapshot call and not just a very long tuple. The arguments to Snapshot should all be indented at least as far as the first space after "Snapshot(" in the first line.

Also, we have been generally using single quotes around strings in oops, rather than double quotes. I do sometimes find some of my old code that used double quotes. I strongly recommend that we standardize on single quotes for all new code. Elsewhere, I tend to leave existing files alone, but I convert double quotes to single when I have some other reason to be working on a file.

Reply:
I thought this was going to be fixed. Please fix.